lib/routes/wordpress/index.ts

```diff
@@ -0,0 +1,163 @@
+import { Route } from '@/types';
+
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import parser from '@/utils/rss-parser';
+import { config } from '@/config';
+
+import { apiSlug, bakeFilterSearchParams, bakeFiltersWithPair, bakeUrl, fetchData, getFilterParamsForUrl, parseFilterStr } from './util';
+
+async function handler(ctx) {
+    const { url = 'https://wordpress.org/news', filter } = ctx.req.param();
+    const limit = ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit'), 10) : 50;
+
+    if (!config.feature.allow_user_supply_unsafe_domain) {
+        throw new Error(`This RSS is disabled unless 'ALLOW_USER_SUPPLY_UNSAFE_DOMAIN' is set to 'true'.`);
```

Comment:
```suggestion
        throw new ConfigNotFoundError(`This RSS is disabled unless 'ALLOW_USER_SUPPLY_UNSAFE_DOMAIN' is set to 'true'.`);
```
https://github.com/DIYgod/RSSHub/blob/984eb4dad4fe4975ecb59eafffa6bd15222eb1da/lib/errors/types/config-not-found.ts#L1-L5

Reply:
😊